net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml

```diff
@@ -16,5 +16,14 @@
       <Required>Y</Required>
       <Multiple>Y</Multiple>
     </interfaces>
+    <blocklist type="NetworkField">
+      <Default></Default>
+      <Required>N</Required>
+      <ValidationMessage>Please enter one or more valid IPv4 networks or individual addresses in CIDR notation.<ValidationMessage>
+      <NetMaskRequired>Y<NetMaskRequired>
```

Comment:
Are you testing what you implement here? The xml is wrong. A few `/` are missing.

Please test what you do.

```
root@opn-ce-01:/src/git/opnsense_plugins/net/mdns-repeater # make lint
/src/git/opnsense_plugins/net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml:27: parser error : Opening and ending tag mismatch: AsList line 26 and blocklist
    </blocklist>
                ^
/src/git/opnsense_plugins/net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml:28: parser error : Opening and ending tag mismatch: AsList line 26 and items
  </items>
          ^
/src/git/opnsense_plugins/net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml:29: parser error : Opening and ending tag mismatch: AddressFamily line 25 and model
</model>
        ^
/src/git/opnsense_plugins/net/mdns-repeater/src/opnsense/mvc/app/models/OPNsense/MDNSRepeater/MDNSRepeater.xml:30: parser error : Premature end of data in tag AddressFamily line 25

^
*** Error code 1

Stop.
```